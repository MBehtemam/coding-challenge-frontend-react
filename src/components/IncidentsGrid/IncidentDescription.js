import styled from 'styled-components'

const IncidentDescription = styled.p`
color:green;
`
export default IncidentDescription