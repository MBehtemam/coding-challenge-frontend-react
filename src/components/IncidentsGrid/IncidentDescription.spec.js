import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import IncidentDescription from './IncidentDescription';

describe('Test suits for Incident Description', () => {
  it('should match with snapshot', () => {
    const tree = renderer
      .create(<IncidentDescription>Some description</IncidentDescription>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
