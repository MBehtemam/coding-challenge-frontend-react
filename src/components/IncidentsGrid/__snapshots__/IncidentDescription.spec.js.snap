// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suits for Incident Description should match with snapshot 1`] = `
.c0 {
  color: green;
}

<p
  className="c0"
>
  Some description
</p>
`;
